nncf/quantization/fake_quantize.py

```diff
@@ -122,11 +122,13 @@ def tune_range(
         fval = -left_border * s
         qval = fns.round(fval)
 
-    ra = fns.where(qval < level_high, qval / (qval - level_high) * right_border, left_border)
```

Comment:
@kshpv, could you briefly explain the reason of these changes?

Reply:
During the run of ptq for `torchvision/mobilenet_v3_small_BC` the runtimewarning appears:
```
RuntimeWarning: invalid value encountered in multiply
    return Tensor(self.data * unwrap_tensor_data(other))
```